src/main/java/sk/uniba/fmph/dcs/stone_age/CivilisationCard.java

```diff
@@ -0,0 +1,4 @@
+package sk.uniba.fmph.dcs.stone_age;
+
+public record CivilisationCard(ImmediateEffect[] immediateEffect, EndOfGameEffect[] endOfGameEffect) { }

```

Comment:
aj tu

Reply:
No nečítal som si pravidlá ale ak to správne chápem tak ten record ma všetky užitočné veci by default v compilatore, teda constructor, gettery,... čiže ak niečo z pravidiel nehovorí ze to ma obsahovať nejake funkcie, alebo tam niečo nenapísal tak je to vpoho podla mňa